connexta/alliance
# alliance-21 updated codebase to reflect new repo/branding

#### What does this PR do?
Update codebase for new branding, refactor package names.
#### Who is reviewing it?
@jlcsmith @kcwire @troymohl @bdeining
#### How should this be tested?
Build against a clean repo.  Extract distro and verify it starts/installs successfully.
#### Any background context you want to provide?
Nope.
#### What are the relevant tickets?
https://github.com/connexta/alliance/issues/21
#### Screenshots (if appropriate)
#### Checklist:
